src/generic/datavgen.cpp

```diff
@@ -3296,7 +3294,7 @@ void wxDataViewMainWindow::RefreshRows( unsigned int from, unsigned int to )
         from = tmp;
     }
 
-    wxRect rect( 0, GetLineStart( from ), GetEndOfLastCol(), GetLineStart( (to-from+1) ) );
+    wxRect rect( 0, GetLineStart( from ), INT_MAX, GetLineStart( (to-from+1) ) );
```

Comment:
BTW, this is unrelated to this commit, but the height of this rectangle is wrong, isn't it? It should be `GetLineStart(to+1) - GetLineStart(from)`, shouldn't it?

Reply:
Yes, thanks for spotting this — I completely missed it there :( The correct expression should be `GetLineStart(to) + GetLineHeight(to) - GetLineStart(from)` (b/c out-of-bounds row passed to `GetLineStart()` doesn't work with variable height).